pkg/network/ovn_kubernetes.go

```diff
@@ -193,7 +193,13 @@ func fillOVNKubernetesDefaults(conf, previous *operv1.NetworkSpec, hostMTU int)
 	// (which may not be a node in the cluster).
 	// However, this can never change, so we always prefer previous.
 	if sc.MTU == nil {
-		var mtu uint32 = uint32(hostMTU) - 100 // 100 byte geneve header
+		var mtu uint32 = uint32(hostMTU) - 50 // 50 byte geneve header
+		for _, cn := range conf.ClusterNetwork {
```

Comment:
ok, based on the comment in https://issues.redhat.com/browse/SDN-1063 from Dan, I am not really sure if this is the best of determining if the cluster is ipv6 or not.

Reply:
@danwinship : PTAL whenever you have some time. The corresponding issue's priority was made high.

Reply:
This is not an urgent fix; if individual customers don't like the default MTU they can change it in their own cluster. (Additionally, based on discussion in other channels, the particular customer who wants this change actually wants it to be 1450, not 1442 anyway, so this won't even help them.)

I don't think we should merge this without also having code to allow changing the MTU in a deployed cluster.

Reply:
+1, I don't think hardcoding MTU would do any good, that said, if there's a need to change the default value as per the following conversation, I'm good for it. But otherwise we'd be changing the hardcoded value as per every now and then customer's request.

Reply:
> +1, I don't think hardcoding MTU would do any good, that said, if there's a need to change the default value as per the following conversation, I'm good for it. But otherwise we'd be changing the hardcoded value as per every now and then customer's request.

agreed, hard-coding default MTU is not something this patch does. I was talking about the header length change.